### PR TITLE
wrong variable name _code => $fail_code

### PR DESCRIPTION
**Pull request by Bischoff**
first case was using $fail_code, others were using _code

switch ... when instead of if ... elsif

better English

### CODE REVIEW

File: features/step_definitions/command_steps.rb [hunk 1]
```diff
@@ -217,20 +217,21 @@ def checkRestart(host, time_out)
...
-  elsif target == "sle-client"
-    _out, _code = $client.run(cmd, false)
+  when "ceos-minion"
+    _out, $fail_code = $ceos_minion.run(cmd, false)
+  when "ssh-minion"
+    _out, $fail_code = $ssh_minion.run(cmd, false)
+  when "sle-minion"
+    _out, $fail_code = $minion.run(cmd, false)
+  when "sle-client"
+    _out, $fail_code = $client.run(cmd, false)
```
Comment (MalloZup) [inline features/step_definitions/command_steps.rb]: we can add here a else statement 

``` else 
 raise "no valid target given"```
for be in the safe side.

It look goods for me, just rebase then after the 3 commits in one

Comment (Bischoff) [inline features/step_definitions/command_steps.rb]: yup.

**Comment by MalloZup** [comment]
@Bischoff , we need also to backport this change to the 3.0 theoretically, if the step is there.

**Comment by MalloZup** [comment]
ok rebase the 3 commits in one and i merge it

**Pull request merged**